Scripts/utils/zone_interval.py

```diff
@@ -140,6 +140,7 @@ def averages(self, array, weights):
             Aggregated array
         """
         aggregation = pandas.Series(index=self.keys)
+        aggregation["all"] = numpy.average(array, weights=weights)
```

Comment:
When `array` and `weights` come from the logit model, they are usually only for "HS15" area. So the "all" aggregation will in this case be for "HS15". The "peripheral" aggregation will in these cases be zero.

Reply:
I see here a chance of misunderstandings... 😕